core-text/src/run.rs

```diff
@@ -83,6 +86,20 @@ impl CTRun {
         }
     }
 
+    pub fn get_typographic_bounds(&self) -> TypographicBounds {
+        let mut ascent = 0.0;
+        let mut descent = 0.0;
+        let mut leading = 0.0;
+        unsafe {
+            let range =  CFRange {
```

Comment:
Nit: Is it Github or the formatting here is slightly off.

Also mind adding a comment here if the range is set to zero the operation will measure from the start index to end of the run?

Reply:
good catch, looks like `rustfmt` wasn't run over the file even before, so did a full format of the file.